### PR TITLE
CIV-6719 Notification Debt Respite Submit Function

### PR DESCRIPTION
**Pull request by jeswanth-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-6719

### Change description ###

Notification Debt Respite Submit Function

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -310,7 +312,7 @@ os-postcode-lookup:
 stitching:
   api:
     url: "${EM_CCD_ORCHESTRATOR_URL:http://localhost:8084}/api/stitch-ccd-bundles"
-  enabled: ${STITCHING_API_ENABLED:true}
+  enabled: ${STITCHING_API_ENABLED:false}
```
Comment (neeta-hmcts) [inline src/main/resources/application.yaml:315]: revert this change

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/BreathingSpaceEnteredLIPNotificationHandler.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            CLAIMANT_NAME, caseData.getApplicant1().getPartyName(),
+            RESPONDENT_NAME, caseData.getRespondent1().getPartyName()
+        );
+    }
+
+    private CallbackResponse notifyApplicant1BreathingSpaceEnteredLip(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        Map<String, String> templateProperties = addProperties(caseData);
+
+        if (isRespondentNotification(callbackParams)) {
```
Comment (neeta-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/BreathingSpaceEnteredLIPNotificationHandler.java:65]: better to add validation for email Id too.

**Comment by jeswanth-hmcts** [comment]
> added minor comments. Plz have a look

resolved the comments

**Review by ElifDenizEsea — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/BreathingSpaceEnteredLIPNotificationHandler.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            String recipientEmail = caseData.getRespondent1() != null ? caseData.getRespondent1().getPartyEmail() : "";
```
Comment (TatsyMaslova) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/BreathingSpaceEnteredLIPNotificationHandler.java:68]: perhaps use Optional.ofNullable here

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/BreathingSpaceEnteredLIPNotificationHandler.java [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            if (recipientEmail != null && !recipientEmail.isEmpty()) {
```
Comment (TatsyMaslova) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/BreathingSpaceEnteredLIPNotificationHandler.java:69]: StringUtils.isNotEmpty() instead?

Comment (jeswanth-hmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/BreathingSpaceEnteredLIPNotificationHandler.java]: used isnotempty

**Review by TatsyMaslova — DISMISSED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]

**Review by neeta-hmcts — DISMISSED** [review]

**Review by laperla11 — APPROVED** [review]

**Review by TatsyMaslova — APPROVED** [review]

**Review by rajakm — APPROVED** [review]
